android/cpp/datasets/README.md

```diff
@@ -6,29 +6,25 @@ TFLite. Other backends may need to extend them.
 
 ## Imagenet
 
-The Imagenet dataset can be downloaded from
-[image-net.org](http://image-net.org/challenges/LSVRC/2012/). The ground truth
-file is
-[imagenet_val.txt](../../java/org/mlperf/inference/assets/imagenet_val.txt)
-which contains indexes of the corresponding class of each images.
-
-If you want to use a subset of images, remember to use the first N ones.
+The full ImageNet dataset is available to MLCommons members individually. 
+You can request access from the mobile chairs.
+For testing purpose, you may download a subset dataset from 
+[https://github.com/mlcommons/mobile_models/tree/main/v0_7/datasets](https://github.com/mlcommons/mobile_models/tree/main/v0_7/datasets)
 
 ## COCO
 
 Download the COCO 2017 dataset from
-[http://cocodataset.org/#download](http://cocodataset.org/#download) and the
-upscale_coco.py from
-[https://github.com/mlperf/inference/blob/master/v0.5/tools/upscale_coco](https://github.com/mlperf/inference/blob/master/v0.5/tools/upscale_coco).
+[http://cocodataset.org/#download](http://cocodataset.org/#download) and the script
+[upscale_coco.py](https://github.com/mlcommons/mobile_app_open/blob/150470388e728894ccb0268b1cef422b410a83e7/mobile_back_qti/datasets/coco/upscale_coco.py).
 Then use the script to process the images:
 
 ```bash
 python upscale_coco.py --inputs /path-to-coco/ --outputs /output-path/ --size 300 300
 ```
 
 The ground truth file is
-[coco_val.pbtxt](../../java/org/mlperf/inference/assets/coco_val.pbtxt) If you
-want to use a subset of images, remember to use the first N images which appear
+[coco_val.pbtxt](https://github.com/mlcommons/mobile_app/blob/c8075ac367554cae98b1508c4a5ab14d14c5885c/java/org/mlperf/inference/assets/coco_val.pbtxt).
```

Comment:
Note that in https://github.com/mlcommons/mobile_models/blob/main/v1_0/assets/tasks_v2.pbtxt the name of the file is expected to coco_val_full.pbtxt

This should likely be fixed so the names are the same.

Reply:
The `coco_val.pbtxt` file in `mobile_app` repo is 7 MB big while the file in `mobile_models` repo is only 6 KB, so I think the ground truth for the full dataset should be the one in `mobile_app` repo.

Reply:
It can't come from mobile_app because that is a private repo. We need a a public source. Perhaps that version should be checked into mobile_models/v1_1/assets/coco_val_full.pbtxt

Reply:
Yes, agreed. I am currently waiting for a confirmation if I can copy the file from `mobile_app` to `mobile_app_open`. See https://github.com/mlcommons/mobile_app_open/issues/73#issuecomment-928920112

Reply:
I uploaded the file to `mobile_models/v1_0/assets/coco_val_full.pbtxt` and created a PR: https://github.com/mlcommons/mobile_models/pull/27.
In `v1_0` because the files `coco_val.pbtxt`, `coco_val_tiny.pbtxt` and `tasks_v2.pbtxt` are there.